lib/ash_oban.ex

```diff
@@ -496,10 +496,10 @@ defmodule AshOban do
           opts =
             case trigger.state do
               :paused ->
-                [events: [paused: true]]
+                [paused: true]
 
               :deleted ->
-                [events: [delete: true]]
+                [delete: true]
```

Comment:
This is for keeping backward-compatibility but I don't see any reason for not making states consistent with Oban (i.e., `delete`).